experimental/dds/sequence-deprecated/src/sparsematrix.ts

```diff
@@ -319,10 +319,7 @@ export class SparseMatrix extends SharedSegmentSequence<MatrixSegment> {
 		const size = maxCols * numRows;
 		const segment = new PaddingSegment(size);
 
-		const insertOp = this.client.insertSegmentLocal(pos, segment);
-		if (insertOp) {
-			this.submitSequenceMessage(insertOp);
-		}
+		this.client.insertSegmentLocal(pos, segment);
```

Comment:
this will still fire and event, which will be caught by the even handler, and send the op before eventing on the sequence